### PR TITLE
small fixes for GrpAbFinGen and TorQuadMod

### PR DESCRIPTION
**Pull request by simonbrandhorst**

### CODE REVIEW

File: test/GrpAb/Map.jl [hunk 1]
```diff
@@ -89,5 +89,12 @@
     h = @inferred hom(G, [3*h for h in gens(H)])
     b = @inferred isbijective(h)
     @test b
+
+    # corner case
+    G = abelian_group(fmpz[])
+    H = abelian_group(fmpz[])
+    i = hom(G, H, gens(H))
+    j = inv(i)
+    @test zero(G)==j(zero(H))
```
Comment (thofma) [inline test/GrpAb/Map.jl:98]:
```suggestion
    @test id(G)==j(id(H))
```
